apps/judicial-system/web/src/routes/Court/Indictments/Conclusion/Conclusion.tsx

```diff
@@ -141,18 +163,26 @@ const Conclusion: React.FC = () => {
       return uploadFiles.find(
         (file) => file.category === CaseFileCategory.COURT_RECORD,
       )
-    }
-
-    if (selectedAction === 'POSTPONE')
+    } else if (selectedAction === 'POSTPONE') {
       return (
         Boolean(
           postponement?.postponedIndefinitely
             ? postponement.reason
             : courtDate?.date,
         ) && allFilesDoneOrError
       )
+    } else if (selectedAction === 'COMPLETE') {
+      return selectedDecision !== undefined && allFilesDoneOrError
+    }
   }
 
+  useEffect(() => {
```

Comment:
Please consider combining this with the `useEffect` above that sets the selected action to `POSTPONED`. Otherwise the two compete for dominance.

Also, consider clearing `indictmentRulingDecision` if the user selects `POSTPONE` after selecting and saving `COMPLETE`. Otherwise, when entering the screen again will show `COMPLETE` selected instead of `POSTPONE`.